scipy/sparse/_lil.py

```diff
@@ -304,6 +304,21 @@ def _set_arrayXarray(self, row, col, x):
                                     self.rows, self.data,
                                     i, j, x)
 
+    def _clear_cells(self, row, col):
+        # 1D
```

Comment:
Can you make this comment a bit more specific?

Reply:
Fixed in 9ec38f609fba370b2f01c81ac7e5ac27b58057db